UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/SceneImporter/ImporterMeshes.cs

```diff
@@ -100,6 +181,136 @@ protected virtual async Task ConstructMesh(GLTFMesh mesh, int meshIndex, Cancell
 			await ConstructUnityMesh(unityData, meshIndex, mesh.Name);
 		}
 
+		/// <summary>
+		/// Populate a UnityEngine.Mesh from Draco generated SubMeshes
+		/// </summary>
+		/// <returns></returns>
+		protected async Task ConstructUnityMesh(GLTFMesh gltfMesh,DracoMeshLoader.DecodeResult[] decodeResults, Mesh.MeshDataArray meshes, int meshIndex, string meshName, bool requireTangents)
```

Comment:
Why is this protected? Shouldn't it be private?
Also please format the parameter list and rename method to something draco-specific